src/components/heading/heading.component.js

```diff
@@ -114,15 +125,16 @@ class Heading extends React.Component {
     }
 
     return (
-      <Link
-        // this event allows an element to be focusable on click event on IE
-        onMouseDown={(e) => e.currentTarget.focus()}
-        className="carbon-heading__back"
-        data-element="back"
-        {...props}
-      >
-        <StyledHeadingIcon type="chevron_left" />
-      </Link>
+      <StyledHeaderLink>
```

Comment:
`StyledHeaderLink` should be a `styled(Link)` so there's no need for the extra wrapping div here

Reply:
wrapper removed